### PR TITLE
OHARA-1394 Change configuration menu title from "Configuration" to "Connection"

### PR DESCRIPTION
**Pull request by DawnJheng**
OHARA-1394 Change configuration menu title from "Configuration" to "Connection"

### CODE REVIEW

**Comment by konekoya** [comment]
run

File: ohara-manager/client/src/components/pages/ConfigurationModal.js [hunk 1]
```diff
@@ -163,7 +163,7 @@ class ConfigurationModal extends React.Component {
             <ConfUl
               isSelected={isDatabaseSelected || isFtpSelected || isHdfsSelected}
             >
-              Configuration
+              Connection
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/ConfigurationModal.js]: Connections :)

Comment (DawnJheng) [reply ohara-manager/client/src/components/pages/ConfigurationModal.js]: OK

**Comment by DawnJheng** [comment]
run

**Comment by konekoya** [comment]
run

**Comment by konekoya** [comment]
retry manager

**Pull request merged**